charts/infra-apps/examples/thanos.yaml

```
@@ -0,0 +1,30 @@
+# Deploy Thanos
+#
+# This takes care of deploying Thanos' store gateway, query frontend, and receiver. 
+# Usually the prometheus-operator from kube-prometheus-stack will deploy 
+# ThanosRulers and sidecars to complete the stack.
+
+thanos:
+  enabled: true
+  values:
+    objstoreConfig:
+      # configure minio client lib for S3 and other S3 compatible endpoints
+      type: S3
+      config:
+        bucket: ""
+        endpoint: ""
+        region: ""
+        access_key: ""
+    # on production, use a secret to not leak S3 credentials
+    # existingObjstoreSecret: mysecret
```

Am now nitpicking my own changes:
```suggestion
      config:
        bucket: ""
        endpoint: ""
        access_key: ""
        secret_key: ""
    # on production, use a secret to not leak S3 credentials
    # existingObjstoreSecret: thanos-objstore-secret
```
Other than this final nitpick, LGTM